docs/tutorials/openvino-extensions.md

```diff
@@ -0,0 +1,337 @@
+
+### FuseML Installer Extension for OVMS Operator
+
+The [official OVMS operator](https://operatorhub.io/operator/ovms-operator) can only be installed with the [Operator Lifecycle Manager (OLM)](https://olm.operatorframework.io/). However, to keep things simple, this implementation uses a Kustomize deployment that installs the OVMS Kubernetes operator that can be tracked down to [this repository](https://github.com/openvinotoolkit/model_server/tree/main/extras/ovms-operator).
+
+Reusing the Kustomize configuration from the official repository is just a matter of writing a `kustomization.yaml` file that points to the remote version:
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - github.com/openvinotoolkit/model_server/extras/ovms-operator/config/default?ref=v2021.4.1
+
+images:
+- name: controller
+  newName: quay.io/openvino/ovms-operator
+  newTag: 0.1.0
+```
+
+Developing a FuseML Installer Extension is done by assembling together a `description.yaml` file with installation instructions and other useful information relevant from an integration perspective. Following are sections from the `description.yaml` descriptor created for the OVMS operator:
+
+* the main section describes the product/version and points to the above Kustomize file for installing/uninstalling the OVMS operator: 
+
+    ```yaml
+    name: ovms
+    product: openvino-model-server
+    version: "0.1.0"
+    description: |
+      OpenVINO Model Server Kubernetes Operator
+    install:
+      - type: kustomize
+        location: kustomize
+        waitfor:
+          - namespace: ovms-operator-system
+            selector: control-plane=controller-manager
+          - kind: deployment
+            selector: control-plane=controller-manager
+            namespace: ovms-operator-system
+            condition: available
+            timeout: 600
+    uninstall:
+      - type: kustomize
+        location: kustomize
+    ```
+
+* adding a `services` section informs FuseML to automatically register the OVMS operator with the FuseML Extension Registry as a service that can be accessed from workflow steps. FuseML workflow steps such as the OVMS predictor, that need the OVMS operator to be preinstalled, will need to reference this extension by specifying an extension requiremenet with a matching `service_resource` field. The _internal_ endpoint URL indicates that the OVMS operator can only be used by workflow steps running in the same Kubernetes cluster (i.e. through a service account). As a side note, it is also be possible to create configurations to allow workflow steps like the OVMS predictor to deploy OVMS instances in remote Kubernetes clusters, provided that Kubernetes credentials are also supplied, but that is not covered here: 
```

Comment:
typo `requiremenet` and there is a trailing space at the end of the line.

Reply:
fixed